app/jobs/reports_uploader_job.rb

```diff
@@ -20,18 +18,11 @@ def perform # rubocop:disable Metrics/AbcSize Layout/LineLength
 
   private
 
-  def upload_submitted_applications
+  def upload_application_details_report
     Rails.logger.info "ReportsUploaderJob - creating submitted applications report at #{Time.zone.now}"
```

Comment:
Would it be worth changing this line to "creating  application_details_report" instead of "creating submitted applications report", for consistency?